src/test/resources/features/CreateEmail.feature

```diff
@@ -0,0 +1,38 @@
+Feature: As a user I want to create a email in my Google Mail account
+
+  Background: User is logged in
+    Given there is a user with valid credentials
+    When the user logs in to Google Mail account
+    Then Home Page opens
+    When the user opens Create New Email page
+    And the user sets Email Addressee = "al.hlebka@gmail.com"
+
+  @bdd
+  Scenario: User creates an email and deletes it
+    When the user sets Email Subject = "AT subject"
+    And the user sets Email Body
+    """
+    Hi!
+    This is At email body.
+    Best Regards,
+    Cucumber
+    """
+    And the user clicks Close New Email button
+    And the user opens Drafts page
+    And the user opens Draft email with Subject = "AT subject"
+    And the user deletes draft email
+    Then Drafts page contains "Нет сохраненных черновиков." header
+
+  @bdd
+  Scenario Outline: User creates an email and sends it
+    When the user sets Email Subject = "<subject>"
```

Comment:
I suppose it would be better to combine the steps with filling in the email data